test/traveler-test.js

```diff
@@ -0,0 +1,131 @@
+import chai from 'chai';
+import Traveler from '../src/traveler';
+import Trip from '../src/trip';
+import dayjs from 'dayjs';
+import Destination from '../src/destination';
+
+const expect = chai.expect;
+
+describe('Traveler', function() {
+    let travelerOneData
+    let travelerTwoData
+    let traveler1
+    let traveler2
+    let tripsData
+    let destinationData
+
+    beforeEach(() => {
+        travelerOneData = {
+            "id": 1,
+            "name": "Ham Leadbeater",
+            "travelerType": "relaxer"
+            };
+        travelerTwoData = {
+            "id": 14,
+            "name": "Ellynn Kyne",
+            "travelerType": "history buff"
+            }
+        traveler1 = new Traveler(travelerOneData)
+        traveler2 = new Traveler(travelerTwoData)
```

Comment:
I'm confused by the need for 2 different travelers for testing here. Any reason for having 2 or can we pass all the tests with just 1?